scripts/camera/orbit-camera.js

```diff
@@ -494,9 +494,9 @@ OrbitCameraInputMouse.prototype.onMouseMove = function (event) {
 
 OrbitCameraInputMouse.prototype.onMouseWheel = function (event) {
     if (this.entity.camera.projection === pc.PROJECTION_PERSPECTIVE) {
-        this.orbitCamera.distance -= event.wheel * this.distanceSensitivity * (this.orbitCamera.distance * 0.1);
+        this.orbitCamera.distance -= event.wheelDelta * -2 * this.distanceSensitivity * (this.orbitCamera.distance * 0.1);
```

Comment:
This page says `WheelEvent.wheelDelta` is deprecated:

https://developer.mozilla.org/en-US/docs/Web/API/WheelEvent

Reply:
The engine seems to use `deltaY` here:

https://github.com/playcanvas/engine/blob/a611fe2ac0da8e89f0adb9d059c91a68a2f2f750/src/platform/input/mouse-event.js#L144-L150

Reply:
Oh wait, this is for a PlayCanvas mouse event. So yep, should be `wheelDeta`. Ignore me!